docs/user/tutorial/50-news.rst

```diff
@@ -66,40 +101,20 @@ Do the same for a second item:
         ride.
 
 
-*********************
-Create an **apphook**
-*********************
-
-Just as we did with Aldryn People, we should create an apphook to create a landing page for the
-news items.
-
-We'll create a *News* page, and add an apphook for the Aldryn News & Blog application to it.
-
-.. todo:: screenshots of following steps (if they are new steps)
-
-.. todo:: write up steps properly
-
-* create and publish a new page called News
-* add Apphook
-* publish page
-* view page with automatic list of news in it
-
-
 .. _use-news-plugin:
 
 *****************
 Use a news plugin
 *****************
 
-.. todo:: screenshots of following steps (if they are new steps)
-
-.. todo:: write up steps properly
-
 * go to home page
 * switch to Edit mode
 * enter Structure mode
 * select **Add plugin** on *Content* placeholder
 * select *Latest articles* plugin from list
-* select the existing Apphook (hopefully we can get rid of this step)
+* select the existing Apphook
 * **Save**
 * see news articles inserted into home page
+
+    .. image:: /user/tutorial/images/home-page-news-articles.png
```

Comment:
If so, I should probably do the same for the people page. Currently I'm only showing the people but not the whole page. @evildmp what do you think?